requirement.txt

```diff
@@ -11,4 +11,5 @@ bitsandbytes==0.41.0
 tokenizers==0.13.3
 accelerate==0.21.0
 llama_cpp_python==0.2.11
-openai-whisper==20231106
\ No newline at end of file
+openai-whisper==20231106
+sqlalchemy==2.0.29
```

Comment:
what's this?

Reply:
SQLAlchemy is a popular SQL toolkit and Object Relational Mapping (ORM) system for Python.
After you update langchain, SQLAlchemy will be a requirement library.

As for openai-whisper, if I want to run app.py in my computer this module is necessary.